lib/views/file-patch-meta-view.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+export default class FilePatchMetaView extends React.Component {
```

Comment:
This classname might confuse me slightly if I hadn't read your guide.

Maybe `FilePatchMetaInfoView` ?  IDK.